cranelift/codegen/src/machinst/compile.rs

```diff
@@ -32,7 +31,7 @@ where
 
     // Creating the vcode string representation may be costly for large functions, so defer its
     // rendering.
-    debug!(
+    log::trace!(
```

Comment:
I think this should remain `debug!`.

Reply:
I think I actually err the other way on this one: the VCode dump is potentially thousands of lines for large functions; including all of the compiled IR in the debug log is potentially much more verbose than the other sorts of entries that people typically generate.